archinstall/lib/menu/global_menu.py

```diff
@@ -42,33 +42,33 @@ def _setup_selection_menu_options(self):
 		# archinstall.Language will not use preset values
 		self._menu_options['archinstall-language'] = \
 			Selector(
-				_('Select Archinstall language'),
+				_('Archinstall language'),
 				lambda x: self._select_archinstall_language(x),
 				default='English')
 		self._menu_options['keyboard-layout'] = \
 			Selector(
-				_('Select keyboard layout'),
+				_('Keyboard layout'),
 				lambda preset: select_language(preset),
 				default='us')
 		self._menu_options['mirror-region'] = \
 			Selector(
-				_('Select mirror region'),
+				_('Mirror region'),
 				lambda preset: select_mirror_regions(preset),
 				display_func=lambda x: list(x.keys()) if x else '[]',
 				default={})
 		self._menu_options['sys-language'] = \
 			Selector(
-				_('Select locale language'),
+				_('Locale language'),
 				lambda preset: select_locale_lang(preset),
 				default='en_US')
 		self._menu_options['sys-encoding'] = \
 			Selector(
-				_('Select locale encoding'),
+				_('Locale encoding'),
 				lambda preset: select_locale_enc(preset),
 				default='UTF-8')
 		self._menu_options['harddrives'] = \
 			Selector(
-				_('Select harddrives'),
+				_('Drives'),
```

Comment:
"Harddrives" maybe? it's what i would look for at least.

Reply:
I thought about "Hard Drives" and "Disk Drives", but then again SSD is not a hdd ;) that's why I ended up with just Drives.
My personal preference would be: **Drive(s)** with parenthesis and all.

Reply:
I might be an old soul, but "Drives" for me represent cars :P Is there anything else we can use? If not I'm ok with this.

Reply:
By "If not I'm ok with this." you mean the new term or the old one? Sorry for the confusion! :D